src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml

```diff
@@ -124,14 +126,14 @@
                                                     Converter={StaticResource NonEmptyStringToCollapsedConverter}}"
                                HorizontalAlignment="Center"
                                Height="18"
-                               Margin="3,0,0,0">
+                               Margin="6,0,0,0">
 
                     </TextBlock>
                 </StackPanel>
 
                 <TextBox Name="SearchTextBox"
                          KeyboardNavigation.TabIndex="0"
-                         Foreground="{StaticResource CommonSidebarTextColor}"
+                         Foreground="#231F20"
```

Comment:
can you replace the string with named versions like existed previously? maybe we'll reuse these somewhere.